app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -744,12 +744,16 @@ class AppointmentRepositoryAndroidTest {
       }
       bpRepository.save(bloodPressureMeasurements).blockingAwait()
 
-      medicalHistoryRepository.save(patientUuid, OngoingMedicalHistoryEntry(
-          hasHadHeartAttack = hasHadHeartAttack,
-          hasHadStroke = hasHadStroke,
-          hasHadKidneyDisease = hasHadKidneyDisease,
-          hasDiabetes = hasDiabetes
-      )).blockingAwait()
+      medicalHistoryRepository.save(
+          uuid = UUID.randomUUID(),
```

Comment:
Can we use an explicit `UUID` here, please?

Reply:
Done.